.pre-commit-config.yaml

```diff
@@ -1,21 +1,22 @@
 repos:
   - repo: local
     hooks:
-      - id: ruff
-        name: ruff
+      - id: ruff-lint
+        name: ruff-lint
         entry: ruff check .
         args: [ --config, pyproject.toml, --fix, --show-fixes, --exit-non-zero-on-fix ]
         language: system
         types_or: [python]
         require_serial: true # run once for all files
         pass_filenames: false
-      - id: black
-        name: black
-        entry: black
+      - id: ruff-format
+        name: ruff-format
+        entry: ruff format .
```

Comment:
I would prefer only formatting the files that are staged, otherwise it might fail if you have other local files

Reply:
Makes sense, but isn't that controlled from the CLI?